.github/workflows/build_wheels_linux.yml

```
@@ -0,0 +1,158 @@
+name: 🐧 Build Linux Wheels
+
+on:
+  workflow_call:
+    inputs:
+      ARCHITECTURE:
+        description: "The architecture to build for"
+        required: true
+        default: "x86_64"
```

actionlint (1.7.11)

[error] input "ARCHITECTURE" of workflow_call event has the default value "x86_64", but it is also required. if an input is marked as required, its default value will never be used

(events)